site/soc/software/charts/osh/openstack-keystone/keystone.yaml

```diff
@@ -27,6 +27,14 @@ data:
     enabled: {{ run_tests }}
     timeout: {{ test_timeout }}
   values:
+    probes:
+      keystone_api:
+        # bump probe timeout to 30s to avoid ingress removing the pods from the pool when they take long to respond
+        # default upstream is 5s
+        readinessProbe:
+          timeoutSeconds: 30
```

Comment:
I think these 30s values are ok for our CI environment due to the performance issues we found and the low amount of replicas we use. For production I don't think these values are going to be acceptable. Waiting 30 seconds to receive an error sounds like too much time.

Since we want to improve our current situation this is a +1 for me, but we do have to remember to tune down these values in the future or make them configurable.

Reply:
totally agree, see incoming PR to make this configurable from the scale profile